sdk/cognitiveservices/cognitiveservices-qnamaker/README.md

```diff
@@ -17,43 +17,28 @@ npm install @azure/cognitiveservices-qnamaker
 
 #### nodejs - Authentication, client creation and getKeys endpointKeys as an example written in TypeScript.
 
-##### Install @azure/ms-rest-nodeauth
-
-```bash
-npm install @azure/ms-rest-nodeauth
-```
-
 ##### Sample code
 
 ```typescript
 import * as msRest from "@azure/ms-rest-js";
 import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
 import { QnAMakerClient, QnAMakerModels, QnAMakerMappers } from "@azure/cognitiveservices-qnamaker";
 const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new QnAMakerClient(creds, subscriptionId);
-  client.endpointKeys.getKeys().then((result) => {
-    console.log("The result is:");
-    console.log(result);
-  });
+const endpoint = "https://westus.api.cognitive.microsoft.com"; // OR some other endpoint.
+const creds = new msRest.ApiKeyCredentials({ inHeader: { 'Ocp-Apim-Subscription-Key': subscriptionId }});
+const client = new QnAMakerClient(creds, endpoint);
+client.endpointKeys.getKeys().then((result) => {
+  console.log("The result is:");
+  console.log(result);
```

Comment:
All the README.md changes are done manually for now. We need to come up with a story to generate samples for data plane.